lib/src/utils/keyMapping.dart

```diff
@@ -43,56 +43,56 @@
 const keyProviderSignInEmailTextField = "keyProviderSignInEmailTextField";
 const keyProviderSignInPasswordTextField = "keyProviderSignInPasswordTextField";
 
-const keyUserProfileEditProfileRaisedButton =
-    "keyUserProfileEditProfilRaisedButton";
-const keyUserProfileShowQrRaisedButton = "keyUserProfileShowQrRaisedButton";
-const keyUserProfileEmailText = "keyUserProfileEmailText";
-const keyUserProfileSettingsAdaptiveIcon= "keyUserProfileSettingsAdaptiveIcon";
+const keyProfileHeaderAdaptiveIconButton = "keyProfileHeaderAdaptiveIconButton";
+const keyProfileHeaderText = "keyProfileHeaderText";
+
+const keyUserProfileSettingsAdaptiveIcon = "keyUserProfileSettingsAdaptiveIcon";
+const keyUserProfileFlagIconSource = "keyUserProfileFlagIconSource";
+const keyUserProfileAvatarIconSource = "keyUserProfileAvatarIconSource";
+const keyUserProfileQrIconSource = "keyUserProfileQrIconSource";
+const keyUserProfileChatIconSource = "keyUserProfileChatIconSource";
+const keyUserProfileSecurityIconSource = "keyUserProfileSecorityIconsource";
+const keyUserProfilePersonAddIconSource = "keyUserProfilePersonAddIconSource";
+const keyUserProfileSignatureIconSource = "keyUserProfileSignatureIconSource";
+const keyUserProfileLockIconSource = "keyUserProfileLockIconSource";
+const keyUserProfileInfoIconSource = "keyUserProfileInfoIconSource";
+const keyUserProfileFeedbackIconSource = "keyUserProfileFeedbackIconSource";
+const keyUserProfileBugReportIconSource = "keyUserProfileBugReportIconSource";
 
 const keyRootIconChatTitleText = "Chat";
 const keyRootIconContactsTitleText = "Contacts";
 const keyRootIconProfileTitleText = "Profile";
 
 const keyUserSettingsCheckIconButton = "keyUserSettingsCheckIconButton";
-const keyUserSettingsUserSettingsUsernameLabel =
-    "key_UserSettingsUserSettingsUsernameLabel";
+const keyUserSettingsUserSettingsUsernameLabel = "key_UserSettingsUserSettingsUsernameLabel";
```

Comment:
```suggestion
const keyUserSettingsUsernameLabel = "key_UserSettingsUsernameLabel";
```
is there a reason that "UserSettings" is twice in the name?